recipes/mintie/mintie-wrapper.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
```

Comment:
Most of what this wrapper does looks like something that should be done in the software itself. As a maintainer or creater of a recipe, I would always try to do as few changes as possible to make the tool work as a Bioconda package. Even if you are both the upstream and the recipe author, it would be good to keep the roles separate.

For example, printing the logo or not is a a decision made upstream, and if the upstream tool doesn’t do it, I wouldn’t add this to a wrapper.

The `downgrade_soap` functionality (if really necessary) is specific to packaging up the tool as a Bioconda package, so I think that could be part of a wrapper, but I would actally prefer if it were a separate script.

Reply:
Thanks for the suggestion. I've removed the wrapper from the recipe and moved it into the software's repo.